recipe-client-addon/bin/make-xpi.sh

```diff
@@ -23,7 +23,7 @@ rm "${DEST}/install.rdf.in"
 cp "${BASE_DIR}/install.rdf" "${DEST}"
 rm "${DEST}/jar.mn"
 cp "${BASE_DIR}/chrome.manifest" "${DEST}"
-rm "${DEST}/test" -r
```

Comment:
No, this change is fine. The way you have it is easier to read anyways. It just took me a moment to realize why it was needed, so I wanted to double check.